randovania/games/am2r/logic_database/GFS Thoth.txt

```diff
@@ -268,8 +268,9 @@ Extra - light_level: 3
           Combat (Advanced) or Normal Damage ≥ 399
           Any of the following:
               # Item requirements
-              Super Missiles ≥ 7 or Combat (Intermediate)
+              Super Missiles ≥ 7 or Combat (Advanced)
               Morph Ball and Power Bombs ≥ 10
+              Long Beam and Combat (Intermediate)
```

Comment:
Genesis with only long beam is intermediate? How many regular shots does it take? unless this is supposed to be in the `all`?

Reply:
this is just what the previous connection was, didn't change it yet. It should probably be higher, dunno how man shots it takes.